code/src/App.js

```diff
@@ -1,9 +1,20 @@
-import React from 'react'
+import React from "react";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import MovieList from "components/MovieList";
+import MovieDetails from "components/MovieDetails";
+import { GlobalStyle } from "styles";
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <>
+      <GlobalStyle />
```

Comment:
Learning from this!

Reply:
We chose this approach so we could remove the style.css file and put the global styles inside our styles.js file